packages/google_sign_in/google_sign_in_web/example/integration_test/auth2_test.dart

```diff
@@ -30,49 +30,50 @@ void main() {
 
   late GoogleSignInPlugin plugin;
 
-  group('plugin.init() throws a catchable exception', () {
+  group('plugin.initialize() throws a catchable exception', () {
```

Comment:
Please, do keep the `init` tests until the method is fully removed from the package.

It's OK to to copy them over to a separate file, like: `auth2_legacy_init_test.dart` or similar. That way we don't lose test coverage in a time that the method will still be used by the core plugin.